app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -195,9 +198,9 @@ class GleanMetricsService(private val context: Context) : MetricsService {
                 code.set(defaultEngine.identifier)
                 name.set(defaultEngine.name)
                 submissionUrl.set(defaultEngine.buildSearchUrl(""))
-
-                Glean.setUploadEnabled(true)
```

Comment:
@hawkinsw I noticed that `Glean.setUploadEnabled` was moved here instead of being called before `Glean.initialize` in #2088 . I'm moving it back in this PR, since this is the [right usage of the API](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean#initializing-glean). Was there a specific reason to move it here? Maybe some bug on the Glean end we need to fix?

Reply:
Nope! Thanks for chatting about this offline and thank you for fixing it!